js/dialog.js

```diff
@@ -28,9 +29,29 @@ export function updateDetailsDialog(data, cardUrl) {
     <p>Artist: ${data["Artist Credit"]} | Writer: ${data["Writer Credit"]}</p>
   `
   );
-  // Card
-  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].src =
-    cardUrl;
+  // Card art is applied
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].style.backgroundImage = 'url("' +
+    cardUrl + '")';
+
+  // Add animation start
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].classList.add(cardRarity(data["Rarity Folder"]), "animated");
+  
+  // Animation reset
+  DETAILS_DIALOG_EL.getElementsByClassName("card-details-dialog-close")[0].addEventListener("click", cardReset);
+  document.getElementsByClassName("dialog-overlay")[0].addEventListener("click", cardReset);
+
+  // animation events
+  let dialogCard = DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0];
+  dialogCard.addEventListener("animationstart", function() {
+    dialogCard.removeEventListener("mouseover", cardAnimate);
+    dialogCard.removeEventListener("click", cardAnimate);
+  }, false);
+  dialogCard.addEventListener("animationend", function() {
+    dialogCard.classList.remove("animated");
+    dialogCard.addEventListener("mouseover", cardAnimate);
+    dialogCard.addEventListener("click", cardAnimate);
+  }, false);
```

Comment:
I'm not sure if it's really best practice or not, but it affects the experience; as the animation will keep stopping and starting. What I'm trying to achieve is a full run of the animation without interruptions from the user; then the opportunity to start it again. :)

Reply:
It looks like the goal here is that you want to prevent the animation from restarting if a user tries to click on the card mid-animation, right? 
You can probably add a class to the dialogCard called `unclickable` and in CSS, define `.unclickable { pointer-events: none; }`. That way we can prevent the click events from firing on animation, while still retaining the event listeners.

Reply:
Ahh just realised because I changed the type of listener I don't think it needs to be done that way anymore; so have updated it.

Reply:
This is updated! :) Thank you to @bonbombs for that CSS suggestion.